test/e2e/common/security_context.go

```diff
@@ -40,6 +41,86 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		podClient = f.PodClient()
 	})
 
+	ginkgo.Context("when creating a pod in the host PID namespace", func() {
+		makeHostPidPod := func(podName, image string, command []string, hostPID bool) *v1.Pod {
+			return &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: podName,
+				},
+				Spec: v1.PodSpec{
+					RestartPolicy: v1.RestartPolicyNever,
+					HostPID:       hostPID,
+					Containers: []v1.Container{
+						{
+							Image:   image,
+							Name:    podName,
+							Command: command,
+						},
+					},
+				},
+			}
+		}
+		createAndWaitHostPidPod := func(podName string, hostPID bool) {
+			podClient.Create(makeHostPidPod(podName,
+				framework.BusyBoxImage,
+				[]string{"sh", "-c", "pidof nginx || true"},
+				hostPID,
+			))
+
+			podClient.WaitForSuccess(podName, framework.PodStartTimeout)
+		}
+
+		nginxPid := ""
+		ginkgo.BeforeEach(func() {
+			nginxPodName := "nginx-hostpid-" + string(uuid.NewUUID())
+			podClient.CreateSync(makeHostPidPod(nginxPodName,
+				imageutils.GetE2EImage(imageutils.Nginx),
+				nil,
+				true,
+			))
+
+			output := f.ExecShellInContainer(nginxPodName, nginxPodName,
+				"cat /var/run/nginx.pid")
+			nginxPid = strings.TrimSpace(output)
+		})
+
+		ginkgo.It("should show its pid in the host PID namespace [LinuxOnly] [NodeFeature:HostAccess]", func() {
```

Comment:
Is `[NodeFeature:HostAccess]` respected in the cluster sweet? I'm actually not sure why it's required though...

Reply:
As I understand it, those tags are just used for regex filtering. So it just depends on if there is some CI tooling that is looking for those tags for inclusion/exclusion.

Reply:
@jefftree can you look into that?

Reply:
@tallclair @johnbelamaric: The exact flag `[NodeFeature:HostAccess]` is not looked at in any of the CI tooling for the cluster suite, but there are a couple of matches for the  `[NodeFeature:.+]`regex (https://github.com/kubernetes/test-infra/search?q=nodefeature%3A.%2B&unscoped_q=nodefeature%3A.%2B).

Reply:
Ok, that looks fine, I don't see any issues there. There are many other `NodeFeature` tests in common already.